index.html

```diff
@@ -116,10 +116,10 @@ <h2>Pairings</h2>
             <div class="pairing-actions">
               <button id="green-button" class="green-button dropdown-toggle">Pair by <span>&#x25B2;</span></button>
                 <div id="toggle-pairing-filter" class="toggle-pairing-filter" style="display: none;">
-                  <button id="same-department">Same Department</button>
-                  <button id="same-location">Same Location</button>
-                  <button id="different-department">Different Department</button>
-                  <button id="different-location">Different Location</button>
+                  <button id="same-department">Same Location</button>
```

Comment:
Aye are these backwards? I see id ="same-department " yet the string is same-locaction. same for all the others

Reply:
good catch!